src/fragments/lib/auth/flutter/signin_next_steps/10_signin.mdx

```diff
@@ -0,0 +1,35 @@
+
+The `Amplify.Auth.signIn` API returns a `SignInResult` object. To see if additional signin steps are required, Inspect the `nextStep.signInStep` property of the result.
+When `result.isSignedIn` is `true` or when the `signInStep` is `done`, the sign in flow is complete.
```

Comment:
I find this some what confusing. What is `result`? Where does `signInStep` live?

Could we denote the relationship relative to`SignInResult`?